util/progress/progressui/printer.go

```diff
@@ -170,10 +170,10 @@ func (p *textMux) printVtx(t *trace, dgst digest.Digest) {
 		p.current = ""
 		v.count = 0
 
+		if v.logsPartial {
+			fmt.Fprintln(p.w, "")
+		}
 		if v.Error != "" {
```

Comment:
My justification for this specific fix is that every single exit path of this `if` chain ends with some variation of `fmt.Fprintf(p.w, "#%d ...`, so it seems correct to make sure that every single path clears any partial line before outputting (even in the case of `CACHED` where I think `logsPartial` shouldn't ever be true).